lib/view/templates.py

```diff
@@ -263,67 +290,103 @@ def usage_weighted_avg(edatas: list[EntryData]):
                 ),
                 Field(
                     "Used%",
-                    Projectors.PercentCompute(
-                        Projectors.Number("ns_stats", "device_used_bytes"),
-                        Projectors.Number("ns_stats", "device_total_bytes"),
+                    Projectors.Div(
+                        Projectors.Number(
+                            "ns_stats", "device_used_bytes", "used-bytes-disk"
+                        ),
+                        Projectors.Number(
+                            "ns_stats", "device_total_bytes", "total-bytes-disk"
+                        ),
                     ),
-                    converter=Converters.round(2),
+                    converter=Converters.ratio_to_pct,
                     aggregator=ComplexAggregator(
                         create_usage_weighted_avg("Device"),
-                        converter=Converters.round(2),
+                        converter=Converters.ratio_to_pct,
                     ),
                     formatters=(
                         Formatters.yellow_alert(
-                            lambda edata: edata.value >= edata.record["Device"]["HWM%"]
+                            lambda edata: edata.value * 100
+                            >= edata.record["Device"]["HWM%"]
                             and edata.record["Device"]["HWM%"] != 0
                         ),
                     ),
                 ),
-                Field("HWM%", Projectors.Number("ns_stats", "high-water-disk-pct")),
+                Field(
+                    "HWM%",
+                    Projectors.Number("ns_stats", "high-water-disk-pct"),
+                    converter=Converters.pct,
+                ),
                 Field(
                     "Avail%",
                     Projectors.Number(
                         "ns_stats", "device_available_pct", "available_pct"
                     ),
+                    converter=Converters.pct,
                     formatters=(Formatters.red_alert(lambda edata: edata.value < 10),),
                 ),
             ),
         ),
         Subgroup(
             "Memory",
             (
+                Field(
+                    "Total",
+                    Projectors.Number("ns_stats", "memory-size", "total-bytes-memory"),
+                    hidden=True,
+                ),
```

Comment:
Why is this hidden?

Reply:
So that it is not displayed. I am following precedence here. If somebody wants the total they can open a ticket

Reply:
The value is needed for the weighted average calculation.  That does not mean it should be displayed